src/index.js

```diff
@@ -243,7 +282,8 @@ function testDataToPost() {
   } else if (roomNumber > roomRepo.rooms.length) {
     displayReservationMessage('room number');
   } else {
-    addNewReservation(date, roomNumber)
+    addNewReservation(date, roomNumber);
+    displayReservationMessage('success')
```

Comment:
I added several reservations but was unable to see a success message and the reservation card stayed there. I refreshed, logged back in and saw that it did in fact reserve it, so the functionality is working! I think there may just be a tweak needed for the display piece.